docs/modules/envoy_access_log_server.md

```diff
@@ -0,0 +1,121 @@
+# Envoy access log server
+
+|                |                        |
+|----------------|------------------------|
+| `moduleName`   | `envoyAccessLogServer` |
+| Module type    | `producer`             |
+| Output event   | `raw`                  |
+
+This module starts a grpcServer ready to accept envoy access logs (see below for supported API version) - it implements [envoy's gRPC ALS](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/accesslog/v3/als.proto#envoy-v3-api-field-service-accesslog-v3-streamaccesslogsmessage-http-logs).
+
+### Envoy support and configuration
+At this moment, V3 of envoy's xDS API is supported. [See the upstream documentation for details](https://www.envoyproxy.io/docs/envoy/latest/api/api_supported_versio) on API versions.
+
+In particular, you have to configure your envoy instance to send access_logs using [v3 AccessLogService rpc](https://github.com/envoyproxy/envoy/blob/842485709d651a6057b2ffb505ffced21173e004/api/envoy/service/accesslog/v3/als.proto#L39). We currently do not implement handling of other versions.
+
+See a minimal example on how to configure envoy to send access_logs to an slo-exporter instance (envoy v1.15+):
+```
+static_resources:
+
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8080
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          access_log:
+          - name: envoy.access_loggers.http_grpc
+            typed_config:
+              "@type": type.googleapis.com/envoy.extensions.access_loggers.grpc.v3.HttpGrpcAccessLogConfig
+              common_config:
+                grpc_service:
+                  envoy_grpc:
+                    cluster_name: service_accesslog
+                buffer_size_bytes:
+                  value: 0
+                log_name: accesslogv3
+                transport_api_version: V3 # needed to ensure that v3.AccessLogService is used
+              additional_request_headers_to_log: ['X-TEST', 'SOMEOHTER-HEADER']
```

Comment:
maybe (e.g. not mandatory) would be nice to show real world exmaple headers, e.g. as 

```suggestion
              additional_request_headers_to_log: ['slo-result', 'slo-class', 'slo-app', 'slo-endponit']
```